databuilder/extractor/bigquery_usage_extractor.py

```diff
@@ -25,12 +25,11 @@ class BigQueryTableUsageExtractor(BaseBigQueryExtractor):
     for referencedTables in the response.
     """
     TIMESTAMP_KEY = 'timestamp'
-    _DEFAULT_SCOPES = ('https://www.googleapis.com/auth/cloud-platform',)
+    _DEFAULT_SCOPES = ['https://www.googleapis.com/auth/cloud-platform']
```

Comment:
['https://www.googleapis.com/auth/cloud-platform',] ?

Reply:
I've previously only seen [trailing commas used for multiline lists](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas) (since it makes the diff prettier), but happy to change if that's the normal style

Either way, I'll look into making flake8 check this -- I'm not partial to the decision, but would like to make sure it's enforced going forward